activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Defining an Enum as a Hash with empty keys now raises an `ArgumentError`.
```

Comment:
Can you amend this this to mention the blank array case?

Reply:
done!